plugins/blockvault_client_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(PkgConfig REQUIRED)
+
+add_library( blockvault_client_plugin
+             blockvault_client_plugin.cpp)
+target_link_libraries( blockvault_client_plugin appbase eosio_chain)
+target_include_directories( blockvault_client_plugin PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" )
+
+
+if (APPLE AND EXISTS "/usr/local/opt/libpq/lib/pkgconfig")
+   # to find libpq installed by homebrew, it is required to set the environment variable PKG_CONFIG_PATH
+   if (DEFINED ENV{PKG_CONFIG_PATH})
+      set(ENV{PKG_CONFIG_PATH} "/usr/local/opt/libpq/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
+   else()
+      set(ENV{PKG_CONFIG_PATH} "/usr/local/opt/libpq/lib/pkgconfig")
+   endif()
+endif(APPLE AND NOT DEFINED ENV{PKG_CONFIG_PATH})
```

Comment:
this mismatch of if()/endif() causes a warning. No need to do this, just use `endif()`